logbook-servlet/src/main/java/org/zalando/logbook/servlet/NormalStrategy.java

```diff
@@ -29,6 +29,8 @@ public void doFilter(final Logbook logbook, final HttpServletRequest httpRequest
             final HttpServletResponse httpResponse, final FilterChain chain) throws ServletException, IOException {
 
         final RemoteRequest request = new RemoteRequest(httpRequest);
+        request.getParameterMap(); // force parameters to be parsed
```

Comment:
This won't allow any subsequent code to read the parameters from the body directly. I'm not sure whether we want to make that decision here.